src/components/Combos/Combos.jsx

```diff
@@ -0,0 +1,265 @@
+import { formatTemplateToString } from '../../utility';
+
+const heroesArray = Object.keys(heroes).map(id => heroes[id]).sort((a, b) => a.localized_name.localeCompare(b.localized_name));
+
+const HeroSelector = ({
+  id,
+  handleHeroSelection,
+  selected,
+  teamAFull,
+  teamBFull,
+  strings,
+  heroName,
+}) => (
+  <StyledHeroSelector>
+    <div className="name-overlay">
+      <div className="name">{heroName}</div>
+      <div className="team-indicator team-a">A</div>
+      <div className="team-indicator team-b">B</div>
+    </div>
+    <HeroImage id={id} />
+    <div className={`ts-container ${selected ? 'selected' : ''}`}>
+      <div
+        className={`ts ts-left ${teamAFull ? 'no-event' : ''}`}
+        onClick={handleHeroSelection(id.toString(), 'teamA')}
+        onKeyPress={handleHeroSelection(id.toString(), 'teamA')}
+        role="button"
+        tabIndex="0"
+      >
+        {formatTemplateToString(strings.add_team, 'A')}
+      </div>
+      <div
+        className={`ts ts-right ${teamBFull ? 'no-event' : ''}`}
+        onClick={handleHeroSelection(id.toString(), 'teamB')}
+        onKeyPress={handleHeroSelection(id.toString(), 'teamB')}
+        role="button"
+        tabIndex="0"
+      >
+        {formatTemplateToString(strings.add_team, 'B')}
+      </div>
+    </div>
+  </StyledHeroSelector>
+);
+
+HeroSelector.propTypes = {
+  id: PropTypes.number,
+  handleHeroSelection: PropTypes.func,
+  selected: PropTypes.bool,
+  teamAFull: PropTypes.bool,
+  teamBFull: PropTypes.bool,
+  heroName: PropTypes.string,
+  strings: PropTypes.shape({}),
+};
+
+const SelectedHeroes = ({
+  teamA, teamB, handleHeroDeSelection, strings,
+}) => (
+  <StyledSelectedHeroes>
+    <div className="team-container">
+      <div className="team-title team-a">{formatTemplateToString(strings.team, 'A')}</div>
+      <div>
+        {[4, 3, 2, 1, 0].map(i =>
+          (teamA[i] ? (
+            <HeroImage
+              id={teamA[i]}
+              className="hero-img"
+              onClick={handleHeroDeSelection(i, 'teamA')}
+            />
+          ) : (
+            <div className="hero-placeholder hero-img" />
+          )))}
+      </div>
+    </div>
+    <div className="seperator">{strings.vs}</div>
+    <div className="team-container">
+      <div className="team-title team-b">{formatTemplateToString(strings.team, 'B')}</div>
+      <div>
+        {[0, 1, 2, 3, 4].map(i =>
+          (teamB[i] ? (
+            <HeroImage
+              id={teamB[i]}
+              className="hero-img"
+              onClick={handleHeroDeSelection(i, 'teamB')}
+            />
+          ) : (
+            <div className="hero-placeholder hero-img" />
+          )))}
+      </div>
+    </div>
+  </StyledSelectedHeroes>
+);
+
+SelectedHeroes.propTypes = {
+  handleHeroDeSelection: PropTypes.func,
+  strings: PropTypes.shape({}),
+  teamA: PropTypes.arrayOf(PropTypes.number),
+  teamB: PropTypes.arrayOf(PropTypes.number),
+};
+
+function asArray(value) {
```

Comment:
This is a fancy trick I used in odota/core to turn things into arrays. It's a bit less readable

`[].concat(input || [])`